### PR TITLE
Remove '<pkg>::' from the response

### LINKED ISSUE #401: `standardize()` fails with `Surv` objects when namespaces are used
As noted by @rempsyc in https://github.com/easystats/report/pull/361

``` r
set.seed(123)
mod_survreg <- survival::survreg(
  formula = survival::Surv(futime, fustat) ~ ecog.ps + rx,
  data = survival::ovarian,
  dist = "logistic"
)

report::report(mod_survreg)
#> Error: Standardization of parameters not possible for models of class 'Surv'.
#>   Try instead to standardize the data (standardize(data)) and refit the
#>   model manually.

> traceback()
23: stop(format_message(string = string, ..., line_length = line_length, 
        indent = indent), call. = call.) at format_message.R#137
22: format_alert(..., type = "error") at format_message.R#155
21: insight::format_error(msg1, "Try instead to standardize the data (standardize(data)) and refit the model manually.")
20: .update_failed(class(x))
19: standardize.Surv(x[[var]], robust = robust, two_sd = two_sd, 
        weights = args$weights, reference = reference[[var]], center = args$center[var], 
        scale = args$scale[var], verbose = FALSE, force = force)
18: standardize(x[[var]], robust = robust, two_sd = two_sd, weights = args$weights, 
        reference = reference[[var]], center = args$center[var], 
        scale = args$scale[var], verbose = FALSE, force = force)
17: standardize.data.frame(data[do_standardize], robust = robust, 
        two_sd = two_sd, weights = if (weights) w, verbose = verbose)
16: standardize(data[do_standardize], robust = robust, two_sd = two_sd, 
        weights = if (weights) w, verbose = verbose)
15: .standardize_models(x, robust = robust, two_sd = two_sd, weights = weights, 
        verbose = verbose, include_response = include_response, update_expr = stats::update(x, 
            data = data_std), ...)
14: standardize.default(model, robust = robust, two_sd = two_sd, 
        include_response = include_response, verbose = verbose, m_info = m_info)
13: datawizard::standardize(model, robust = robust, two_sd = two_sd, 
        include_response = include_response, verbose = verbose, m_info = m_info)
12: standardize_parameters.default(model, ...)
11: parameters::standardize_parameters(model, ...)
10: effectsize.default(x, method = effectsize_method, ...)
9: effectsize::effectsize(x, method = effectsize_method, ...)
8: withCallingHandlers(expr, warning = function(w) if (inherits(w, 
       classes)) tryInvokeRestart("muffleWarning"))
7: suppressWarnings(effectsize::effectsize(x, method = effectsize_method, 
       ...))
6: report_effectsize.survreg(x, ...)
5: report_effectsize(x, ...)
4: report_table.survreg(x, include_effectsize = include_effectsize, 
       effectsize_method = effectsize_method, ...)
3: report_table(x, include_effectsize = include_effectsize, effectsize_method = effectsize_method, 
       ...)
2: report.survreg(mod_survreg)
1: report::report(mod_survreg)
```

<sup>Created on 2023-04-03 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

``` r
library(survival, quietly = TRUE, warn.conflicts = FALSE)

set.seed(123)
mod_survreg <- survreg(
  formula = Surv(futime, fustat) ~ ecog.ps + rx,
  data = ovarian,
  dist = "logistic"
)

report::report(mod_survreg)
#> Can't calculate log-loss.
#> Can't calculate proper scoring rules for models without integer response values.
#> `performance_pcp()` only works for models with binary response values.
#> Can't calculate log-loss.
#> Can't calculate proper scoring rules for models without integer response values.
#> `performance_pcp()` only works for models with binary response values.
#> We fitted a logistic model to predict Surv(futime, fustat) with ecog.ps and rx
#> (formula: Surv(futime, fustat) ~ ecog.ps + rx). The model's explanatory power
#> is weak (Nagelkerke's R2 = 0.07). The model's intercept, corresponding to
#> ecog.ps = 0 and rx = 0, is at 667.43 (95% CI [-415.59, 1750.45], p = 0.227).
#> Within this model:
#> 
#>   - The effect of ecog ps is statistically non-significant and negative (beta =
#> -210.59, 95% CI [-726.18, 305.01], p = 0.423; Std. beta = -107.06, 95% CI
#> [-369.19, 155.07])
#>   - The effect of rx is statistically non-significant and positive (beta =
#> 320.10, 95% CI [-194.11, 834.32], p = 0.222; Std. beta = 163.22, 95% CI
#> [-98.98, 425.42])
#>   - The effect of Log(scale) is statistically significant and positive (beta =
#> 5.82, 95% CI [5.35, 6.29], p < .001; Std. beta = 5.82, 95% CI [5.35, 6.29])
#> 
#> Standardized parameters were obtained by fitting the model on a standardized
#> version of the dataset. 95% Confidence Intervals (CIs) and p-values were
#> computed using a Wald z-distribution approximation.
```

<sup>Created on 2023-04-03 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

### PR DESCRIPTION
**Pull request by etiennebacher**
Close #401 

``` r
set.seed(123)
mod_survreg <- survival::survreg(
  formula = survival::Surv(futime, fustat) ~ ecog.ps + rx,
  data = survival::ovarian,
  dist = "logistic"
)

report::report(mod_survreg)
#> Can't calculate log-loss.
#> Can't calculate proper scoring rules for models without integer response values.
#> `performance_pcp()` only works for models with binary response values.
#> Can't calculate log-loss.
#> Can't calculate proper scoring rules for models without integer response values.
#> `performance_pcp()` only works for models with binary response values.
#> We fitted a logistic model to predict survival::Surv(futime, fustat) with
#> ecog.ps and rx (formula: survival::Surv(futime, fustat) ~ ecog.ps + rx). The
#> model's explanatory power is weak (Nagelkerke's R2 = 0.07). The model's
#> intercept, corresponding to ecog.ps = 0 and rx = 0, is at 667.43 (95% CI
#> [-415.59, 1750.45], p = 0.227). Within this model:
#> 
#>   - The effect of ecog ps is statistically non-significant and negative (beta =
#> -210.59, 95% CI [-726.18, 305.01], p = 0.423; Std. beta = -107.06, 95% CI
#> [-369.19, 155.07])
#>   - The effect of rx is statistically non-significant and positive (beta =
#> 320.10, 95% CI [-194.11, 834.32], p = 0.222; Std. beta = 163.22, 95% CI
#> [-98.98, 425.42])
#>   - The effect of Log(scale) is statistically significant and positive (beta =
#> 5.82, 95% CI [5.35, 6.29], p < .001; Std. beta = 5.82, 95% CI [5.35, 6.29])
#> 
#> Standardized parameters were obtained by fitting the model on a standardized
#> version of the dataset. 95% Confidence Intervals (CIs) and p-values were
#> computed using a Wald z-distribution approximation.
```

### CODE REVIEW

File: R/standardize.models.R [hunk 1]
```diff
@@ -123,9 +123,19 @@ standardize.default <- function(x,
...
   if (!include_response) {
-    resp <- unique(c(insight::find_response(x), insight::find_response(x, combine = FALSE)))
+    resp <- unique(
+      c(
+        insight::clean_names(insight::find_response(x)),
+        insight::clean_names(insight::find_response(x, combine = FALSE))
+      )
+    )
   } else if (include_response && two_sd) {
-    resp <- unique(c(insight::find_response(x), insight::find_response(x, combine = FALSE)))
```
Comment (etiennebacher) [inline R/standardize.models.R:128]: @strengejacke these expressions are identical in the `if` and in the `else`, is this expected?

Comment (strengejacke) [reply R/standardize.models.R:128]: Hm, doesn't look right. @mattansb you also worked on this function, any ideas?

Comment (mattansb) [reply R/standardize.models.R:128]: I'll have to take a closer look in a few days... But basically any change to this file will break everything :(

Comment (mattansb) [reply R/standardize.models.R:128]: Cleaned up the code to prevent duplicate lines

Comment (etiennebacher) [reply R/standardize.models.R:128]: Thanks

**Comment by etiennebacher** [comment]
@strengejacke can I merge this?

**Comment by strengejacke** [comment]
Yes, I think so. @mattansb has checked the code, right?

**Review by strengejacke — APPROVED** [review]

**Comment by etiennebacher** [comment]
> Yes, I think so. @mattansb has checked the code, right?

Yes he fixed it

**Pull request merged**